Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
@OskarStark basically we changed our position about this a few months ago (I cannot find the link to the exact discussion). We no longer care about `@covers` and `@codeCoverageIgnore` because in our opinion is not worth it to maintain them. After all, we don't follow TDD or want to achieve 100% coverage, so it's not that important for us.